app/ui-flextab/client/tabs/userActions.js

```diff
@@ -49,12 +50,12 @@ export const getActions = ({ user, directActions, hideAdminControls }) => {
 		user && user._id && !!RoomRoles.findOne({ rid: Session.get('openedRoom'), 'u._id': user._id, roles: 'moderator' });
 	const isInDirectMessageRoom = () => {
 		const room = ChatRoom.findOne(Session.get('openedRoom'));
-		return (room && room.t) === 'd';
+		return roomTypes.getConfig(room.t).allowMemberAction(room, RoomMemberActions.BLOCK);
```

Comment:
Can you rename the function to something better like `canBlockRoom`?

Reply:
In fact, this method isn't necessary anymore since you are already checking exactly the same thing after calling this.

Reply:
What about this one?